src/main/webapp/app/entities/flair-bi/flair-bi.html

```diff
@@ -180,10 +188,12 @@
                     </div>
                     <div ng-style="{'opacity': v.bodyProperties.opacity}"
                         ng-if="$root.vizualizationServiceMode == 'grpc'" id="content-{{v.visualBuildId}}"
-                        class="printme content widget-content" visualization-render-grpc can-build="v.canBuild()" data="v" id="v.id || v.visualBuildId" widget="{{v.metadataVisual.functionname}}"
+                        class="printme content widget-content" visualization-render-grpc can-build="v.canBuild()"
+                        data="v" id="v.id || v.visualBuildId" widget="{{v.metadataVisual.functionname}}"
                         reload="vm.change" drilldowns="vm.drilldowns" datasource="vm.datasource">
                         <div class="loader-spinner" ng-if="v.metadataVisual.functionname==='GenerateTable'">
-                            <img class="vizualization" src="../../../content/svgs/loader.svg" style="height: 100px;width: 100px;">
+                            <img class="vizualization" src="../../../content/svgs/loader.svg"
+                                style="height: 100px;width: 100px;">
```

Comment:
Convert the svg tag to symbol and place css in scss file.. We need get rid of svg from system and convert them into symbols